client/blocks/site/index.jsx

```diff
@@ -88,7 +88,7 @@ export default React.createClass( {
 		return (
 			<div className={ siteClass }>
 				<a className="site__content"
-					href={ this.props.homeLink ? site.URL : this.props.href }
+					href={ this.props.homeLink ? site.slug : this.props.href }
```

Comment:
I don't understand this change. Slug doesn't seem desirable as the anchor `href`. Also probably breaks pretty badly on subfolder installs (where slug is e.g. `example.com::foo` if I recall correctly).